docker/sagemaker/Dockerfile.sm

```diff
@@ -44,23 +44,23 @@ RUN pip3 install \
 
 
 # Copy workaround script for incorrect hostname
-COPY build_artifacts/changehostname.c /opt/ml/code/
+COPY build_artifacts/changehostname.c /usr/local/bin/changehostname.c
 COPY build_artifacts/start_with_right_hostname.sh /usr/local/bin/start_with_right_hostname.sh
 
-# /opt/ml and all subdirectories are utilized by SageMaker, we use the /code subdirectory to store our user code.
-# TODO(xiangsx): change to pip install when PIP package is available
-COPY code/graphstorm/ /opt/ml/code/graphstorm/
-ENV PYTHONPATH="/opt/ml/code/graphstorm/python/:${PYTHONPATH}"
+# /opt/ml and all subdirectories are utilized by SageMaker,
+# we use the /usr/local/lib subdirectory to store our user code.
+COPY code/graphstorm/ /usr/local/lib/graphstorm/
+ENV PYTHONPATH="/usr/local/lib/graphstorm/python/:${PYTHONPATH}"
 
-RUN cp /opt/ml/code/graphstorm/sagemaker/run/* /opt/ml/code/
+RUN cp /usr/local/lib/graphstorm/sagemaker/run/* /usr/local/lib/
```

Comment:
For the WORKDIR, we can still use `/opt/ml/code/`.

In the future, we may use `pip install graphstorm` to replace L51 and L53.